docs/development/release_management.rst

```diff
@@ -8,23 +8,46 @@ must be done by people that have special privileges to do specific tasks
 but even if the **Release Manager** does not have those privileges, they should
 coordinate with the person that does to make sure the task is completed.
 
+In addition to the Release Manager, we typically recognize the following
+roles for a SecureDrop release:
+
+- **Deputy RM:** for additional time zone coverage, to delegate specific tasks,
+  and to act as backup in case of the RM becomes unavailable for any reason.
+
+- **Localization Manager:** to manage outreach to the translator community, and
+  to coordinate translation updates of existing strings.
+
+- **Deputy LM:** like the RM, this role is backed up by another team member.
+
+- **Communications Manager:** to prepare and distribute pre-release and
+  release messaging (including standard upgrade instructions, release notes,
+  social media posts, and support portal announcements)
+
+During the full release cycle, we also recognize the following role:
```

Comment:
Wondering if there should be a designated QA Manager who can give daily updates to the RM or deputy RM about bugs or issues the QA team is having, especially since there could be timezone differences where the RM might not be able to follow along as it happens?

Reply:
I've suggested that previously as well, but I think for now we should just capture current practices, and discuss this a bit more & potentially try it with 1.5.0. Does that make sense?

Reply:
that sounds good to me